server/app_env/integration_test.py

```diff
@@ -20,4 +20,10 @@ class Config(_base.Config):
   API_ROOT = 'https://autopush.api.datacommons.org'
```

Comment:
should make this AutopushConfig then?

Reply:
Done.

Reply:
Also update below to be StagingConfig(base._Config): with 5 fields defined?

Reply:
Thanks!  Indeed, I badly broke tests.